plugins/inputs/bigbluebutton/README.md

```diff
@@ -0,0 +1,44 @@
+# BigBlueButton Input Plugin
+
+The BigBlueButton Input Plugin gathers metrics from [BigBlueButton](https://bigbluebutton.org/) server. It uses [BigBlueButton API](https://docs.bigbluebutton.org/dev/api.html) getMeetings and getRecordings endpoints to query the data.
```

Comment:
I think emphasis the endpoints like
```suggestion
The BigBlueButton Input Plugin gathers metrics from [BigBlueButton](https://bigbluebutton.org/) server. It uses [BigBlueButton API](https://docs.bigbluebutton.org/dev/api.html) `getMeetings` and `getRecordings` endpoints to query the data.
```
makes it a bit easier to read. But only cosmetics.

Reply:
Done

> @SLedunois Thank you very much for submitting this nice plugin! The code looks quite good, but I have some comments nevertheless. :-) My major points are
> 
> 1. Please check errors! Especially for the XML unmarshalling!
> 2. There are too many indirections in the code that makes it hard to follow the program flow. Please directly call the final function (e.g. `acc.AddFields()`) instead of call a three level deep stack where each call is just a rename of the previous one. :-)
> 3. Simplify your code! I put some examples as comments, but there are more I think. Check again and boil it down to the essence.
> 
> Last but not least I suggest you merge `api.go` back to `bigbluebutton.go` in order to have all calls in one place. Do _not_ merge `model.go`. This might be cosmetic but it will ease the review of the code.

Hi @srebhan 

Thanks for the review. I fix my code based on your comments.

Reply:
Done